README.md

```diff
@@ -68,6 +68,12 @@ password  | String | Cloudant password
 url       | String | Cloudant URL containing both username and password
 modelIndex | String | Specify the model name to document mapping, defaults to 'loopback\_\_model\_\_name'
 
+**NOTE: Please be aware of that the URL will override username/password and must follow a [right format](https://github.com/cloudant/nodejs-cloudant#initialization)**:
```

Comment:
I would omit mentioning the format of the URL; the required format is a result of the URI standard.

Additionally, I would *not* mention the downstream issue in our comments, since it will ideally be fixed soon. Putting those comments in will just require removal later.